### PR TITLE
Update Vue editors demos to composition API style

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: JSDemos/Demos/Autocomplete/Overview/Vue/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
     <link rel="stylesheet" type="text/css" href="../../../../../node_modules/devextreme-dist/css/dx.light.css" />
 
+    <script src="../../../../../node_modules/typescript/lib/typescript.js"></script>
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Autocomplete/Overview/Vue/index.html:10]: Why do we add ts as script? Could we add ts to SystemJS config?

File: JSDemos/Demos/CheckBox/Overview/Vue/App.vue [hunk 2]
```diff
@@ -79,42 +79,31 @@
...
+const handleValueChangeLabel = computed(() => {
+  return { 'aria-label': 'Handle value change' };
+});
+const disabledLabel = computed(() => {
+  return { 'aria-label': 'Disabled' };
+});
+const customSizeLabel = computed(() => {
+  return { 'aria-label': 'Custom size' };
+});
+</script>
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/CheckBox/Overview/Vue/App.vue]: Add new lines

File: JSDemos/Demos/Autocomplete/Overview/Vue/index.html (hunk 1 above)
Comment (GoodDayForSurf) [reply JSDemos/Demos/Autocomplete/Overview/Vue/index.html:10]: I tried to do so, but without success

**Comment by alexanderPolosatov** [comment]
Let's do not create PRs with equal names(name is equal with [this PR](https://github.com/DevExpress/devextreme-demos/pull/2712))

**Pull request closed without merging**